alicloud/provider.go

```diff
@@ -363,6 +363,7 @@ func Provider() terraform.ResourceProvider {
 			"alicloud_ots_instance":                        resourceAlicloudOtsInstance(),
 			"alicloud_ots_instance_attachment":             resourceAlicloudOtsInstanceAttachment(),
 			"alicloud_cms_alarm":                           resourceAlicloudCmsAlarm(),
+			"alicloud_cms_sitemonitor":                     resourceAlicloudCmsSiteMonitor(),
```

Comment:
I think the `alicloud_cms_site_monitor` will be better.

Reply:
done

Reply:
done